### ISSUE TITLE
Bats tests fail on test 9

### ISSUE THREAD

**Issue by fancyremarker** [aptible/docker-kibana#3]
cc: @mwenger1

Can't repro locally but opening for further future review

```
not ok 9 docker-kibana protects all pages with basic auth
# (in test file /tmp/test/kibana.bats, line 57)
#   `[ \"$status\" -eq 0 ]' failed
# Adding password for user root
# main: SIGTERM (SignalException)
# rm: cannot remove '/var/log/nginx/access.log': No such file or directory
# rm: cannot remove '/var/log/nginx/error.log': No such file or directory
```

**Comment by krallin** [comment]
I ran into this as well; this is a timing issue where `timeout 1` exits before `run-kibana.sh` has had time to launch Nginx. This will be fixed by #13 by building on Travis (i.e. customers won't run the tests anymore when deploying).

**Comment by krallin** [comment]
Fixed in #13
